src/codeGenerators/circuit/zokrates/toCircuit.ts

```diff
@@ -8,6 +8,14 @@ import NodePath from '../../../traverse/NodePath.js'
 import {traversePathsFast} from '../../../traverse/traverse.js'
 const Circuitbp = new CircuitBP();
 
+const removeTrailingSemicolon = (code: string) => {
```

Comment:
We may be able to get rid of these helpers by adding the semicolons to expression statements.